app/ide-desktop/lib/dashboard/src/authentication/src/authentication/cognito.ts

```diff
@@ -34,8 +34,27 @@ import * as cognito from 'amazon-cognito-identity-js'
 import * as results from 'ts-results'
 
 import * as config from './config'
+import * as loggerProvider from '../providers/logger'
 import * as platformModule from '../platform'
 
+// =================
+// === Constants ===
+// =================
+
+/** String used to identify the GitHub federated identity provider in AWS Amplify.
+ *
+ * This provider alone requires a string because it is not a standard provider, and thus has no
+ * constant defined in the AWS Amplify library. */
+const GITHUB_PROVIDER = 'Github'
+
+const MESSAGES = {
+    signInWithPassword: {
+        userNotFound: 'User not found. Please register first.',
```

Comment:
I was writing about "user not found" in previous PR review. Please be a little bit more precise. I believe that we are searching for user name / login, not for the user itself ;)